core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -204,6 +235,8 @@ object AdminClient {
   val DefaultSendBufferBytes = 128 * 1024
   val DefaultReceiveBufferBytes = 32 * 1024
   val DefaultRetryBackoffMs = 100
+  val DefaultGroupQueryRetryIntervalMs: Int = 100
```

Comment:
I agree, we could reuse the other var. Please go ahead with the fix. Thanks a lot.